openpype/vendor/python/common/qargparse.py

```diff
@@ -629,7 +629,9 @@ def set_current(current):
                     )
 
             qindex = model.index(index, 0, QtCore.QModelIndex())
-            smodel.setCurrentIndex(qindex, smodel.ClearAndSelect)
```

Comment:
@iLLiCiTiT an you confirm these changes are ok?

Reply:
Yes.